### DISCUSSION TITLE
C# ingester SDK

### DISCUSSION THREAD

**Discussion by killme2008** [GreptimeTeam/greptimedb#3717] [Ideas]
### What problem does the new feature solve?

We already have Java, Golang, JavaScript, and Erlang ingester, but still missing c# sdk for .net platform.

I think it's better to provide it too.

### What does the feature do?

Implement an ingester SDK in c# programing language.

**Comment by tisonkun** [comment]
Let's move this topic to General discussions. It's not something can be done in the main repo.
